website/docs/workloads/mlperf/mlperf-profiles.md

```diff
@@ -36,6 +36,7 @@ Runs the MLPerf benchmark workload to test GPU performance.
   | LinuxCudaVersion     | Optional. Version of CUDA that needs to be installed. | 11.6  |
   | LinuxDriverVersion     | Optional. Version of GPU driver that needs to be installed. | 510  |
   | LinuxLocalRunFile     | Optional. Link to download specified CUDA and GPU driver versions. | https://developer.download.nvidia.com/compute/cuda/12.0.0/local_installers/cuda_12.0.0_525.60.13_linux.run  |
+  | RequireCustomSystemSupport | Optional. This enables A100_PCIe_80GBx4 system support that was not supported by github repo of MLPerf. | true |
```

Comment:
Could you expand on this a bit more? It's not clear it
1. We only support A100*4
2. Only on A100 we need this
3. Only on A100*4 we need this

Reply:
Is it better now? We are providing additional this system support, original systems which open source support are supported already.